cpu/lpc1768/periph/uart.c

```diff
@@ -157,9 +157,11 @@ void uart_write(uart_t uart, const uint8_t *data, size_t len)
             return;
     }
 
-    for (size_t i = 0; i < len; i++) {
-        while (!(dev->LSR & (1 << 5)));       /* wait for THRE bit to be set */
-        dev->THR = data[i];
+    if(dev) {
```

Comment:
missing space before parentheses

```suggestion
    if (dev) {
```

Reply:
amend directly, please

Reply:
sorry. Just pushed the amended commit